gym-unity/gym_unity/__init__.py

```diff
@@ -1,5 +1,5 @@
 # Version of the library that will be used to upload to pypi
-__version__ = "0.24.0.dev0"
+__version__ = "0.24.0"
 
 # Git tag that will be checked to determine whether to trigger upload to pypi
 __release_tag__ = None
```

Comment:
Make sure to use the script that the release checklist says - that should set the release tag here (and you'll have to update a bunch of doc links once that happens).